libs/langchain-community/src/vectorstores/mongodb_atlas.ts

```diff
@@ -61,6 +80,9 @@ export class MongoDBAtlasVectorSearch extends VectorStore {
     this.indexName = args.indexName ?? "default";
     this.textKey = args.textKey ?? "text";
     this.embeddingKey = args.embeddingKey ?? "embedding";
+    this.primaryKey = args.primaryKey ?? "_id";
+    this.overwrite = args.overwrite ?? false;
```

Comment:
Default for most vectorstores is actually upsert - would be nice to get this to match at some point

Reply:
Can do, I'll try to push it some time tomorrow.